server/controllers/users/searchUsers.js

```diff
@@ -0,0 +1,19 @@
+const { users: { searchUsers } } = require('../../database/queries');
+
+exports.searchUsers = (req, res, next) => {
+  // we will use req.query to get results based on it
+  const { search: searchQuery } = req.query;
+  searchUsers(searchQuery)
+    .then(({ rows }) => {
+      if (rows.length && rows) {
```

Comment:
```suggestion
      if ( rows[0] ) {
```

Reply:
if rows is null or undefined, `rows[0]` will throw an error. You might argue that the searchUsers method never returns undefined or null, but that might change in the future (for example if you upgrade the underlying dependency for querying the DB). It is better to program defensively, I would keep the check (just turn it around as mentioned before)

Reply:
@11janci. oh thx  to notify me for this point